runtime/doc/builtin.txt

```diff
@@ -1169,6 +1171,43 @@ autocmd_get([{opts}])					*autocmd_get()*
 		Return type: list<dict<any>>
 
 
+base64_decode({string})					*base64_decode()*
+		Return a Blob containing the bytes decoded from the base64
+		characters in {string}.
+
+		The {string} argument should contain only base64-encoded
+		characters and should have a length that is a multiple of 4.
+
+		Returns an empty blob on error.
+
+		Examples: >
+		    " Write the decoded contents to a binary file
+		    call writefile(base64_decode(s), 'tools.bmp', 'B')
```

Comment:
Is it the flag 'B' a typo meaning 'b'? I have not find any reference to the flag 'B' of writefile. And to my suprise, it can be any string: `call writefile(['/tmp/a'], '/tmp/a', 'ZZZZZZZ')`.

Reply:
The 'B' flag is accepted by the readfile() function and not by the writefile() function.  I will update the example to remove the 'B' flag.